package.json

```diff
@@ -102,6 +108,7 @@
     "pixrem": "^4.0.1",
     "postcss-banner": "^3.0.0",
     "serve-favicon": "^2.4.2",
+    "sigmund": "^1.0.1",
```

Comment:
Not sure we need sigmund as an explicit dependency.

Reply:
same as for each-async